e3sm_diags/driver/utils/dataset_xr.py

```diff
@@ -385,17 +387,22 @@ def _get_climo_dataset(self, season: str) -> xr.Dataset:
         filepath = self._get_climo_filepath(season)
         ds = self._open_climo_dataset(filepath)
 
-        if self.var in ds.variables:
-            pass
-        elif self.var in self.derived_vars_map:
+        if self.var in self.derived_vars_map and (self.var,) in list(
+            self.derived_vars_map[self.var].keys()
+        ):
             ds = self._get_dataset_with_derived_climo_var(ds)
```

Comment:
```suggestion
	if self.var in self.derived_vars_map:
```

I noticed you pushed the above change in [`daa748e` (#752)](https://github.com/E3SM-Project/e3sm_diags/pull/752/commits/daa748e06953b34db1fb76c8a67f5df1446183d6) but reverted it in [`7a494ef` (#752)](https://github.com/E3SM-Project/e3sm_diags/pull/752/commits/7a494ef514062981ad38ba4ad41ef3b365c486f8#diff-35006a05805f6f5876c9fe70b5281c865955b8a82d1574fd7f62d6efda623975).

Was this on accident? This explains why the CI/CD build is still breaking.

Also are you certain you are still running into issues 3 and 4 in this [comment](https://github.com/E3SM-Project/e3sm_diags/pull/752#issuecomment-1927819504) with this update? A re-test might be a good idea.